media-fonts/iosevka/iosevka-1.14.1.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit font
+VERSION="1.14.1"
```

Comment:
Why are you introducing a custom variable for it? Version is already provided as PV.